packages/localizations/src/en-US.ts

```diff
@@ -2,6 +2,26 @@ import type { LocalizationResource } from '@clerk/types';
 
 export const enUS: LocalizationResource = {
   locale: 'en-US',
+  taskSelectOrganization: {
+    signOut: {
+      actionLink: 'Sign out',
+      actionText: 'Signed in as {{identifier}}',
+    },
+    createOrganization: {
+      title: 'Setup your account',
+      subtitle: 'Tell us a bit about your organization',
+      formButtonSubmit: 'Create organization',
+      formButtonReset: 'Cancel',
+    },
+    selectOrganization: {
+      title: 'Choose an account',
```

Comment:
```suggestion
      title: 'Choose your organization',
```

We're going to update the copy here - it's wrong, there's no personal accounts being displayed in this component.